packages/gatsby-transformer-remark/src/extend-node-type.js

```diff
@@ -403,12 +408,14 @@ module.exports = (
     return resolve({
       html: {
         type: GraphQLString,
+        workerPlugin,
```

Comment:
Not sure I like that. This will break graphql-js/graphql-compose typings for people who use typescript. Could we do smth like 

```js
const workerPlugin = new GatsbyWorkerPlugin(`gatsby-transformer-remark`)

{
   htmlAst: {
        type: GraphQLJSON,
        resolve: workerPlugin.wrapResolver((markdownNode) => {
          // ...
        },
     }
}

Reply:
Love this idea. It's way more explicit rather than flagging and picking it up later in the process. Thanks!

Reply:
Hmm, the implementation to get something like `wrapResolver` is gonna be pretty hacky. But it will work. Does GraphQL have any other way of specifying "tags" on Field objects? I essentially want to attach metadata to a field.